### ISSUE TITLE
why the name prop compulsory?

### ISSUE THREAD

**Issue by canercak** [toystars/react-native-multiple-select#140]
using "label" as "displaykey" doesn't work. Why make "name" prop is compulsory? If it's compulsory why "displaykey" exists? The standard is always "value"-"label" set.

**Comment by koiralakiran1** [comment]
Yes. This is also something that I also noticed. It totally ignores the `displayKey` property when used. Took me about an hour to figure this out. Please update the documentation or the functionality. :)
